TESTING-INSTRUCTIONS.md

```diff
@@ -1,5 +1,49 @@
 # Testing instructions
 
+## Unreleased
```

Comment:
This should be 2.8.0

Reply:
Oh, I thought that this was a change that should be done during the release.
I changed it in the commit 4705274